Client/src/components/App.js

```diff
@@ -1,9 +1,12 @@
-import React from 'react'
+import React from 'react';
+import Mobile from './Layout/Mobile';
+// import Descktop from './Layout/Desktop';
 
 export default function App() {
   return (
     <div>
-        Hello World
+      <Mobile />
+      {/* <Descktop /> */}
```

Comment:
Comment Here !!! ?

Reply:
this comment for other team members they work on the Desktop layout